src/tests/dbus-tests/test_10_basic.py

```diff
@@ -6,7 +6,7 @@
 class UdisksBaseTest(udiskstestcase.UdisksTestCase):
     '''This is a base test suite'''
 
-    udisks_modules = set(['Bcache', 'BTRFS', 'ISCSI.Initiator', 'LVM2', 'ZRAM'])
+    udisks_modules = set(['Bcache', 'BTRFS', 'ISCSI.Initiator', 'LVM2', 'ZRAM', 'VDO'])
 
     def setUp(self):
         self.manager_obj = self.get_object('/Manager')
```

Comment:
Please update also the `_get_modules` function below so the `test_20_enable_modules` test doesn't fail on systems without the vdo module.

Reply:
Fixed.